Module3_1/Module3_1/Program.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+namespace Module3_1
+{
+    public class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("введите первое число: ");
+            int a = int.Parse(Console.ReadLine());
+            Console.WriteLine("введите второе число: ");
+            int b = int.Parse(Console.ReadLine());
+            int result = 0;
+            if (a < 0 && b < 0)
+                
+                for (int i = 0; i < -a; i++)
+                {
+                    result += -b;
+                }
+            
+            else if (a > 0 && b > 0)
+                
+                for (int i = 0; i < a; i++)
+                {
+                    result += b;
+                }
```

Comment:
DRY (don't repeat yourself)
you already used it on 15 line - you need to think how to just reuse it

what about:
            var isNegative = a < 0 && b > 0 || a > 0 && b < 0;

            for (int i = 0; i < Math.Abs(Math.Min(a, b)); i++)
            {
                result += Math.Abs(Math.Max(a, b));
            }
Console.WriteLine(isNegative ? -result : result);